tests/scala/org/pantsbuild/native_image/FindMacrosSuite.scala

```diff
@@ -0,0 +1,147 @@
+package org.pantsbuild.native_image
+
+import scala.meta.testkit.DiffAssertions
+import org.scalatest.FunSuite
+import coursier._
+import java.io.File
+
+class FindMacrosSuite extends FunSuite with DiffAssertions {
+  test("expect") {
```

Comment:
I think it might be worth using `ignore` for this test since it downloads quite a lot of jars from Maven Central and I estimate the `FindMacros` class won't change so frequently.